code/frameworks/nextjs/src/utils.ts

```diff
@@ -94,7 +94,15 @@ export const addScopedAlias = (baseConfig: WebpackConfig, name: string, alias?:
  * scopedResolve('styled-jsx') === '/some/path/node_modules/styled-jsx'
  */
 export const scopedResolve = (id: string): string => {
-  const scopedModulePath = require.resolve(id, { paths: [path.resolve()] });
+  // id = css-loader/package.json
+  let scopedModulePath;
+
+  try {
+    scopedModulePath = require.resolve(id, { paths: [path.resolve()] });
+  } catch (e) {
+    scopedModulePath = require.resolve(id);
+  }
```

Comment:
So that was to fix for pnp right, we just left the top statement for backwards compat.
We should likely add a todo here, for ourselves, to just have the bottom statement in 8.0?